secure_message/resources/messages.py

```diff
@@ -18,6 +20,7 @@
 
 logger = wrap_logger(logging.getLogger(__name__))
 
+MESSAGES_SERVICE_ERROR = "Messages service error"
```

Comment:
As above re: consistency. This is also not a super descriptive error message; if we can give any more detail we probably should.

Reply:
Updated as above